### DISCUSSION TITLE
Invalid SFTP packet size

### DISCUSSION THREAD

**Discussion by mazzy89** [blinksh/blink#1843] [Support] [answered]
### Checklist

- [X] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [X] Read the [docs](http://blink.sh/docs/)
- [X] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Describe the bug

Trying to use `code` command to connect to my workspace but I get the following error in the log

```
[error] Apr 29 2023, 19:47:18 Connection ::1.49647 Not handled error - connError(msg: "Invalid sftp packet size!")
[info] Apr 29 2023, 19:47:31 FileSystem De-registering mount 1
``` 

Code shows my file system in the remote machine

**Comment by mazzy89** [comment]
Very likely the issue is at the level of the blink-fa extension

**Comment by carloscabanero** [comment]
Hi! I honestly have never seen this one before. It looks like a mismatch at either side of the protocol. What ssh server are you using? Any specific Linux distro?

Blink-fs works on top of sftp, could you try to do a regular `sftp —verbose file remote:`?

**Comment by mazzy89** [comment]
The server is a MacOS. Gotta pull down the ssh version for you. Sure let me gather this info for you

**Comment by carloscabanero** [comment]
Yep, although being macOS, unless you are running a special openssh from homebrew or connecting to an old versiom, not sure we will get much info.

Would you mind trying the verbose and see what we get from it? There isn’t much online about this error either…

**Comment by mazzy89** [comment]
This is the current version of openssh

```

ssh -V
OpenSSH_9.0p1, LibreSSL 3.3.6
```

Let me run that command for you

**Comment by mazzy89** [comment]
I was able to replicate the issue w/ running your command

```
sftp --verbose .blink/blinkCode.log myremotehost:
Copy failed. connError(msg: "Invalid sftp packet size!")
```

No much info more than this. The only thing i can try is to run the command against another host.

**Comment by mazzy89** [comment, marked as answer]
I've checked and here the results.
I've tested in two other different hosts one MacOS and one Linux and the command works. It does not work only in this other MacOS host so something is off there. Well at this point does not seem a bug in the product so we can close this and i can try to investigate on my own why this MacOS hosts fails sftp. 
Thanks for the help here

**Comment by carloscabanero** [comment]
Hmmmm… It can still be some quirk on Libssh, that maybe for whatever reason is not liking that folder, server, etc.. How about trying to log under a different user to the remote? And maybe trying to install a different openssh version through Homebrew?

This is definitely just some quirk between client - server. Hard to pin point it though, but hopefully trying things will give us clues.

**Comment by mazzy89** [comment]
No luck unfortunately. Installed in the remote machine openssh from home brew to the latest version. Used a different user and still the same error.

**Comment by carloscabanero** [comment]
Moving as a Discussion in case someone else has a similar issue.
